### PR TITLE
updated seneka_windsensor

### PR DESCRIPTION
**Pull request by dabertram**
added parameters;
rewrote code for extracting sensor values and message publishing

### CODE REVIEW

File: seneka_windsensor/common/include/seneka_windsensor/SerialIO.h [hunk 1]
```diff
@@ -12,12 +12,22 @@
...
  * Date of creation: Jan 2013
+ * Date of modification: Oct 2013
+ *
  * ToDo:
+ * -- clean up
+ * -- restructure code
+ * -- test
  *
+ * ToDo - extra features:
+ * ++ writing to windsensor/ setting windsensor-internal parameters possible?
```
Comment (mgruhler) [inline seneka_windsensor/common/include/seneka_windsensor/SerialIO.h]: What about all those ToDo's?
Still to be done or are they completed/not possible?

If so, please remove

File: seneka_windsensor/common/src/SerialIO.cpp [hunk 2]
```diff
@@ -68,352 +70,279 @@
...
-	//	printf("%2d Bytes read:", BytesRead);
-	//	for(int i=0; i<BytesRead; i++)
-	//	{
-	//		unsigned char uc = (unsigned char)Buffer[i];
-	//		printf(" %u", (unsigned int) uc );
-	//	}
-	//	printf("\n");
+// ################################################################################################################################
+// stuff below is NOT USED for windsensor   (..yet)
+// ################################################################################################################################
```
Comment (mgruhler) [inline seneka_windsensor/common/src/SerialIO.cpp]: What about this stuff?
If unnecessary, please remove to get a cleaner status

**Comment by dabertram** [comment]
removed those comments;
code should be "clean" now, leaving some unused functions in serialIO.cpp (writing to serial) as they might be useful later

**Comment by dabertram** [comment]
not building anymore.. fixing it

**Pull request merged**